### PR TITLE
Update author, licensing

### LINKED ISSUE #6: Update plugin author, url
We should get the data updated and also check if the updater code also needs updating.

### PR DESCRIPTION
**Pull request by robincornett**
Fixes #6 

Proposed Changes:
1. Deleted obsolete plugin updater class.
2. Update plugin author.
3. Update licensing initialization.

### CODE REVIEW

File: edd-campaign-monitor.php [hunk 1]
```diff
@@ -23,9 +25,19 @@
...
 
-if( class_exists( 'EDD_License' ) ) {
-	$edd_cm_license = new EDD_License( __FILE__, 'Campaign Monitor', '1.1.1', 'Sandhills Development, LLC', 'eddcp_license_key', null, 974 );
-}
+/**
+ * Sets up licensing with EDD core.
+ * @todo When minimum EDD version is >= 2.11.4, remove checks.
+ */
+add_action( 'plugins_loaded', function () {
+	if ( class_exists( '\\EDD\\Extensions\\ExtensionRegistry' ) ) {
```
Comment (ashleyfae) [inline edd-campaign-monitor.php:33]: :heart_eyes:

File: edd-campaign-monitor.php [hunk 2]
```diff
@@ -23,9 +25,19 @@
 |--------------------------------------------------------------------------
 */
 
-if( class_exists( 'EDD_License' ) ) {
-	$edd_cm_license = new EDD_License( __FILE__, 'Campaign Monitor', '1.1.1', 'Sandhills Development, LLC', 'eddcp_license_key', null, 974 );
-}
+/**
+ * Sets up licensing with EDD core.
+ * @todo When minimum EDD version is >= 2.11.4, remove checks.
```
Comment (ashleyfae) [inline edd-campaign-monitor.php:30]: Also :heart_eyes:

File: edd-campaign-monitor.php [hunk 3]
```diff
@@ -23,9 +25,19 @@
...
+ */
+add_action( 'plugins_loaded', function () {
+	if ( class_exists( '\\EDD\\Extensions\\ExtensionRegistry' ) ) {
+		add_action( 'edd_extension_license_init', function( \EDD\Extensions\ExtensionRegistry $registry ) {
+			$registry->addExtension( __FILE__, 'Campaign Monitor', 974, '1.1.1', 'eddcp_license_key' );
```
Comment (ashleyfae) [inline edd-campaign-monitor.php:35]: Can we define a constant for the version number? Just so we don't have to remember to update it in both places.

Comment (robincornett) [reply edd-campaign-monitor.php]: Done. Set to `EDDCP_VERSION` but I confess I don't know why everything in this extension uses `CP`.

Comment (ashleyfae) [reply edd-campaign-monitor.php]: CamPaign.
Cam... Pain...

Just brainstorming.

**Review by ashleyfae — APPROVED** [review]

**Pull request merged**